test/test_dht_storage.cpp

```diff
@@ -83,29 +83,39 @@ namespace
 		g_storage_constructor_invoked = true;
 		return dht_default_storage_constructor(settings);
 	}
+
+	std::unique_ptr<dht_storage_interface> create_default_dht_storage(
+		dht_settings const& sett)
+	{
+		std::unique_ptr<dht_storage_interface> s(dht_default_storage_constructor(sett));
+		TEST_CHECK(s.get() != NULL);
```

Comment:
for future reference, we should use nullptr instead of NULL. no need to go through another CI cycle for that now though